app/app.js

```diff
@@ -969,19 +950,22 @@ var AppComponent = React.createClass({
     this.redirectToDefaultRoute();
     trackMetric('Signup Verified');
   },
-
   handleAcceptedTerms: function() {
-    this.setState({
-      showingAcceptTerms: false
+    var self = this;
+    var acceptedDate = sundial.utcDateString();
+    app.api.user.acceptTerms({ terms: acceptedDate },function(err) {
+      if(_.isEmpty(err)){
```

Comment:
This was a little hard to read, since we're usually doing `if (err) { handle error }` whereas this is the opposite. Was there a specific reason you wanted handling the error to be the default action here?